tests/cbmc/proofs/s2n_array_free/s2n_array_free_harness.c

```diff
@@ -33,6 +33,7 @@ void s2n_array_free_harness()
 
 #pragma CPROVER check push
 #pragma CPROVER check disable "pointer"
+#pragma CPROVER check disable "pointer-overflow"
```

Comment:
As discussed in #2774, we should remove these `#pragma`s in future.

So for now you could just leave this unchanged (CBMC 5.15 is anyway happy with arithmetic on NULL).

Reply:
Implemented as suggested.